src/java.se/share/data/jdwp/jdwp.spec

```diff
@@ -2150,8 +2150,6 @@ JDWP "Java(tm) Debug Wire Protocol"
             (Error THREAD_NOT_SUSPENDED)
             (Error OPAQUE_FRAME      "Attempted to return early from a frame "
                                      "corresponding to a native method, "
-                                     "the thread is a suspended virtual thread and the target "
-                                     "VM is unable to force its current frame to return, "
                                      "or the implementation is unable to provide this "
                                      "functionality on this frame.")
```

Comment:
This is okay but I wonder if you've considered nudging it closer to wording in JVMTI ForceEarlyReturnXXX so it would say the implementation is unable to force the frame to return and use a native frame as an example.